spec/fixtures/hiera/hiera.yaml

```diff
@@ -1,8 +1,10 @@
 ---
-:backends:
-  - yaml
-:yaml:
-  :datadir: 'spec/fixtures/hiera'
-:hierarchy:
-  - fqdn/%{fqdn}
-  - parameter_tests/%{parameter_tests}
+version: 5
+defaults:
+  datadir: data          # This path is relative to hiera.yaml's directory.
```

Comment:
Wouldn't this keep it from using the data in fixtures which is the point of this file?

Reply:
No, sorry the `spec` tests are configured to use this `spec/fixtures/hiera/hiera.yaml` and the `datadir` is relative to that file's directory, so it uses: `spec/fixtures/hiera/data`. Does that make sense? Sorry for the confusing wording.